README.md

```diff
@@ -116,8 +116,145 @@ An example layer file in the path `/layers/ken-health-sites.json`
 }
 
 ```
+Gisida supports various data sources. These include: <br>
+**csv -**  A [csv](https://en.wikipedia.org/wiki/Comma-separated_values) file can be used solely as a source without joining it with any other source as long as it has geolocations columns i.e longitude and latitude. If the fields are named differently one will need to include geo-columns property on the layer spec as shown below. 
+``` 
+"geo-columns": ["longitude", "latitude"] 
+``` 
+An example of such a layer
+```json
+{
+  "label": "Kenya Health Sites",
+  "source": {
+    "type": "geojson",
+    "featureType": "Point",
+    "data": "data/ken_health_sites.csv"
+  },
+  "geo-columns": ["longs", "lats"],
+  "type": "symbol",
+  "minZoom": 0,
+  "paint": {
+    "text-color": "#000",
+    "text-halo-color": "#fff",
+    "text-halo-width": 1.3,
+    "text-halo-blur": 1
+  },
+  "layout": {
+    "text-field": "{name} ({type})",
+    "text-offset": [0,2],
+    "icon-image": "hospital-11",
+    "icon-allow-overlap": true,
+    "text-transform": "uppercase"
+  },
+  "visible": false,
+  "credit": "Global Healthsites Mapping Project<br>Aug 15, 2017"
+} 
+```
+We can also join the csv with a vector layer being served from mapbox. To achieve this we need to add a columns with simmillar data on both datasets for the join to work.
+```
+"join":["name","name"]
```

Comment:
One thing that I always have to think twice about is which one is which, maybe we can change this to be something like `"join: ["vectorProp", "csvProp"]`?